MdsSupportingClasses/MdsColliveryService.php

```diff
@@ -762,7 +762,8 @@ public function orderToCollivery(WC_Order $order, array $overrides) {
         if (isset($overrides['collection_time']) && $overrides['collection_time']) {
             $colliveryOptions['collection_time'] = $overrides['collection_time'];
         } else {
-            $collectionTime = date('Y-m-d H:i:s', strtotime(date('Y-m-d').' + 1 days + 12 hours'));
+            $leadTime = $this->settings->getValue('lead_time') ?? 24;
```

Comment:
You've changed the default/fall-back from 36 hours to 24, why?

Also please note that, if `$this->settings->getValue('lead_time')` returns `0` then it will not fall-back to `24`
I think that maybe you meant to use `?:` here?
`??` = `isset()`
`?:` = `(bool) === false`

Generally speaking `?:` is the way to go unless you're not sure that the data structure you're accessing is valid. Like `$foo['bar']['baz']` or `$foo->bar->baz`

Reply:
36 hours is too long.
If 0 is set then the lead_time must be 0 not the fall back